index.html

```diff
@@ -144,7 +144,7 @@ <h1>
       });
     </script>
 
-    <a-scene overlay-visibility>
+    <a-scene renderer="colorManagement: false" overlay-visibility>
```

Comment:
I think we should make work with the default colorManagment?

Reply:
I'll leave it out so that it uses the default value (`false` on 1.4.2 and `true` on master). Probably a remnant from my testing, as I switched between 1.4.2 and master quite often and wanted to compare apples to apples.